src/main/java/org/gridsuite/study/server/dto/ReportInfos.java

```diff
@@ -17,6 +18,7 @@
  */
 @AllArgsConstructor
 @Getter
+@EqualsAndHashCode
 @Schema(description = "Report infos")
 public class ReportInfos {
```

Comment:
This could be turned into a record : 
```suggestion
@Schema(description = "Report infos")
public record ReportInfos(
        UUID reportUuid,
        String reporterId
) {
}
```

Reply:
DONE